include/cucumber-cpp/internal/CukeDll.hpp

```diff
@@ -0,0 +1,14 @@
+#ifndef CUKE_CUKEDLL_HPP_
+#define CUKE_CUKEDLL_HPP_
+
+#ifndef CUKE_API_
```

Comment:
Don't do this check, it should be considered a bug (and thus reported by the compiler) if it's already defined.